proarc-common/src/main/java/cz/cas/lib/proarc/common/imports/TiffImporter.java

```diff
@@ -67,10 +67,10 @@
  *
  * @author Jan Pokorsky
  */
-public final class TiffImporter {
+public class TiffImporter implements ImageImporter {
```

Comment:
Tohle kromě loggeru asi neplatí. Neuvědomil jsem si, že je to varianta `jpeg/jp2k->tiff->...`